python/pyspark/ml/_typing.pyi

```diff
@@ -64,7 +65,7 @@ MultilabelClassificationEvaluatorMetricType = Union[
     Literal["microRecall"],
     Literal["microF1Measure"],
 ]
-ClusteringEvaluatorMetricType = Union[Literal["silhouette"]]
+ClusteringEvaluatorMetricType = Literal["silhouette"]
```

Comment:
Not sure what the `Union` here was for.

Reply:
This was intentional to make it consistent with the remaining `*EvaluatorMetricTypes`

Reply:
Shall I revert this change, then? My editor was complaining about the `Union` since it only had one member (and I agree with my editor in this case, but it's not a big deal to me).

Reply:
I am OK with this either way, just giving the context.